### PR TITLE
Add device Xiaomi Redmi note 5 supported

### PR DESCRIPTION
**Pull request by TTAsphyxia**

### CODE REVIEW

File: data/devices/xiaomi/04_redmi_note.yml [hunk 1]
```diff
@@ -1,4 +1,36 @@
 ---
+- name: Xiaomi Redmi note 5
+  id: whyred
+
+  codenames:
+    - whyred
+    - Redmi Note 5
+
+  architecture: Kryo 260
```
Comment (chenxiaolong) [inline data/devices/xiaomi/04_redmi_note.yml]: The `architecture` field should be `armeabi-v7a` (if 32-bit) or `arm64-v8a` (if 64-bit).

File: data/devices/xiaomi/04_redmi_note.yml [hunk 2]
```diff
@@ -1,4 +1,36 @@
...
+
+    base_dirs:
+      -/dev/block/platform/soc/c0c4000.sdhci/by-name
```
Comment (chenxiaolong) [inline data/devices/xiaomi/04_redmi_note.yml]: Missing space character.

**Comment by yshalsager** [comment]
@chenxiaolong Devices with vendor partition is supported now?

**Comment by chenxiaolong** [comment]
@yshalsager No, Treble devices (with vendor partitions) are still not supported yet.

**Comment by chenxiaolong** [comment]
@noobdev-bot test this please

**Pull request merged**